dask/dataframe/tests/test_groupby.py

```diff
@@ -2187,3 +2187,64 @@ def test_groupby_transform_ufunc_partitioning(npartitions, indexed):
                 lambda series: series - series.mean()
             ),
         )
+
+
+@pytest.mark.parametrize("dropna", [False, True, None])
+def test_groupby_dropna_pandas(dropna):
+
+    # The `dropna` arg is not currently supported by pandas
+    # (See #https://github.com/pandas-dev/pandas/pull/21669)
+    # Dask supports the argument for the cudf backend,
+    # but should raise an error for pandas.
+
+    # TODO: Expand test when `dropna`is supported in pandas.
+
+    df = pd.DataFrame(
+        {"a": [1, 2, 3, 4, None, None, 7, 8], "e": [4, 5, 6, 3, 2, 1, 0, 0]}
+    )
+    ddf = dd.from_pandas(df, npartitions=3)
+
+    if dropna is None:
+        dask_result = ddf.groupby("a").e.sum()
+        pd_result = df.groupby("a").e.sum()
+        assert_eq(dask_result, pd_result)
+    else:
+        with pytest.raises(TypeError):
+            ddf.groupby("a", dropna=dropna)
+        with pytest.raises(TypeError):
+            df.groupby("a", dropna=dropna)
+
+
+@pytest.mark.parametrize("dropna", [False, True, None])
+@pytest.mark.parametrize("by", ["a", "c", "d", ["a", "b"], ["a", "c"], ["a", "d"]])
+def test_groupby_dropna_cudf(dropna, by):
```

Comment:
@mrocklin @TomAugspurger I imagine that we *don't* want a cudf-specific test like this in dask,  but do you have any thoughts on a `dropna` test that would be covered by CI?  Is `test_groupby_dropna_pandas` sufficient?

Reply:
Happy to keep this here I think. I imagine that someday we could have a GPU-enabled job in our CI.